mbtiles/__init__.py

```diff
@@ -44,11 +45,25 @@ def process_tile(tile):
         *mercantile.ul(tile.x + 1, tile.y + 1, tile.z))
 
     kwds = base_kwds.copy()
-    kwds['transform'] = from_bounds(ulx, lry, lrx, uly, 256, 256)
+    kwds['transform'] = transform_from_bounds(ulx, lry, lrx, uly, 256, 256)
```

Comment:
@jdmcbr can you generalize this to 256 or 512 tile sizes?

Reply:
@sgillies I'd be happy to just do #33 in a separate PR, if that is okay.

Reply:
That is, I started towards generalizing tile sizes here, and then felt like it made more sense to just go all the way and add the new user option all in one go, but then I thought this PR would start to be a bit of a hydra.